src/Blorc.OpenIdConnect/Models/Profile.cs

```diff
@@ -1,10 +1,45 @@
 ﻿namespace Blorc.OpenIdConnect
 {
+    using System.Collections.Generic;
     using System.Security.Claims;
+    using System.Text.Json;
     using System.Text.Json.Serialization;
 
     public class Profile
     {
+        [ClaimType("aud")]
+        [JsonPropertyName("aud")]
+        public JsonElement Aud
+        {
+            get;
+            set;
+        } = JsonDocument.Parse("[]").RootElement;
+
+        public string[] Audiences
+        {
+            get
+            {
+                if (Aud.ValueKind == JsonValueKind.Array)
+                {
+                    var result = new List<string>();
+                    using var enumerator = Aud.EnumerateArray();
+
+                    foreach (var item in enumerator)
+                    {
+                        result.Add(item.GetString()!);
```

Comment:
It might not be possible to have a "null" value in the array, but even then it could be wise to add a null check:

```
var itemString = item.GetString();
if (itemString is not null)
{
    result.Add(itemString);
}
```